CHANGES.md

```diff
@@ -33,6 +33,7 @@ Release Notes.
 * The namespace and cluster would be reported as instance properties, keys are `namespace` and `cluster`. Notice, if
   instance_properties_json includes these two keys, they would be overrided by the agent core.
 * [Breaking Change] Remove the namespace from `cross process propagation` key.
+* Make sure the parent endpoint in tracing context from existing first ENTRY span, rather than first span only.
```

Comment:
I feel tough to understand the meaning of this sentence without reading the codes.

```suggestion
* Make sure the parent endpoint in tracing context is the first ENTRY span, rather than first span only.
```

Reply:
`is from` is not correct, because this trace context could only have local/exit spans.